pkgs/tools/networking/dd-agent/default.nix

```diff
@@ -38,6 +38,8 @@ in stdenv.mkDerivation rec {
     sha256 = "1iqxvgpsqibqw3vk79158l2pnb6y4pjhjp2d6724lm5rpz4825lx";
   };
 
+  patches = [ ./40103-iostat-fix.patch ];
```

Comment:
Can you use fetchpatch of this commit instead? https://github.com/DataDog/dd-agent/commit/bfc3057fa68b2d6bbc223780c3933e893a6ba38f

Reply:
I'll test that

Reply:
@Mic92 The diff of the test does not apply cleanly (`dd-agent` is at a *much* newer version upstream than what we have packaged). Not sure if it's worth pursuing that vs. having the patch in nixpkgs for a short while (I'll follow up with a proper fix by updating the package version eventually, see #39808)

Reply:
@Mic92 Currently the datadog integration in 18.03 is broken without this change. Can you please merge this into 18.03? I had expected that this change would be done simultaneously with the other change, because both are required for a working system.